immudb/client.py

```diff
@@ -28,48 +32,101 @@
 from immudb.embedded.store import KVMetadata
 import threading
 import queue
+import immudb.datatypesv2 as datatypesv2
+import immudb.dataconverter as dataconverter
 
 import datetime
 
+from immudb.streamsutils import KeyHeader, StreamReader, ValueChunk, ValueChunkHeader, BufferedStreamReader
+
 
 class ImmudbClient:
 
-    def __init__(self, immudUrl=None, rs: RootService = None, publicKeyFile: str = None, timeout=None):
+    def __init__(self, immudUrl=None, rs: RootService = None, publicKeyFile: str = None, timeout=None, max_grpc_message_length=None):
+        """immudb Client
+
+        Args:
+            immudbUrl (str, optional): url in format ``host:port``
+                (e.g. ``localhost:3322``) of your immudb instance.
+                Defaults to ``localhost:3322`` when no value is set.
+            rs (RootService, optional): object that implements RootService,
+                allowing requests to be verified. Optional.
```

Comment:
If not provided, a default in-memory RootService instance will be created.

Reply:
Resolved in 07780c9